lib/util/http/index.ts

```diff
@@ -126,6 +126,14 @@ async function gotTask<T>(
 }
 
 export class Http<Opts extends HttpOptions = HttpOptions> {
+  private static defaultConcurrentRequestLimit: number | null = null;
+  private static defaultMaxRequestsPerSecond: number | null = null;
+
+  static setDefaultLimits(): void {
+    Http.defaultConcurrentRequestLimit = 5;
+    Http.defaultMaxRequestsPerSecond = 5;
+  }
+
```

Comment:
This part is ugly, but it helps to avoid rate limits during test runs. Actually, it seems not to be a huge slow down, but I'm afraid it'll add up over time.